### PR TITLE
Make modelhub demo notebook work with BQ

### PR DESCRIPTION
**Pull request by thijs-obj**
resolves issue https://github.com/objectiv/objectiv-analytics/issues/951

### CODE REVIEW

File: notebooks/model-hub-demo-notebook.ipynb [hunk 1]
```diff
@@ -327,45 +327,31 @@
...
-   "id": "c7522e41",
-   "metadata": {},
-   "source": [
-    "### 3. Further data crunching using the Bach Modeling Library\n",
-    "All results from the model hub are in the form of Bach DataFrames or Series. This makes the model hub and Bach work seamlessly together."
+    "# We'll do a lot of operations on the data in the `df` DataFrame. To make this easier for the database (especially\n",
+    "# BigQuery), we tell Bach to materialize the current DataFrame as temporary table. This statement has no direct\n",
+    "# effect, but any invocation of head() on the dataframe later on will consist of two queries: one to create a\n",
+    "# temporary table with the current state of `df`, and one that queries that table and does following operations\n",
+    "df = df.materialize(materialization='temp_table')"
```
Comment (thijs-obj) [inline notebooks/model-hub-demo-notebook.ipynb:348]: Seems like the easiest way to make this work

**Comment by vincenthoogsteder** [comment]
> from my side the changes are ok. maybe @vincenthoogsteder 's call to say if we want to remove metabase in favor of bigquery support for the entire notebook.

@hendrik-obj yes I'm all good removing metabase completely from the notebooks & docker demo. We have a better story now with Hex etc and can explain easily that folks can use the SQL in Metabase and others.

**Comment by thijs-obj** [comment]
> if we are ok on the changes we do still need to change https://github.com/objectiv/objectiv-analytics/blob/main/bach/docs/source/example-notebooks/modelhub-basics.rst accordingly.


Good one

**Comment by thijs-obj** [comment]
Updated the website documentation too. Can you have a look if this is OK @vincenthoogsteder @vard-obj ?

**Review by vincenthoogsteder — APPROVED** [review]
LGTM

**Pull request merged**